js/controllers/grid/articleGalleys/ArticleGalleyGridHandler.js

```diff
@@ -58,11 +58,18 @@
 		// FIXME: Inter-widget messaging is needed here.
 		var selector = 'a[id^="component-grid-articlegalleys-articlegalleygrid-row-' +
 				rowId + '-addFile-button-"]';
-		$.when($(selector)).then(function() {
-			$(function() {
+
+		function defferedClick(selector) {
```

Comment:
`deferred` rather than `deffered` (spelling)

Reply:
:+1: I knew that looked funny! Fixed now.